crypto-common/src/lib.rs

```diff
@@ -370,3 +368,19 @@ impl fmt::Display for InvalidLength {
 
 #[cfg(feature = "std")]
 impl std::error::Error for InvalidLength {}
+
+/// The error type returned when a random number generator fails.
+#[cfg(any(feature = "getrandom", feature = "rand_core"))]
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+pub struct RngError;
```

Comment:
Potentially it could carry along an inner code e.g. the `raw_os_error` as reported by either `getrandom::Error` or `rand_core::Error`, although I don't think it's bad for it to be completely opaque